Cargo.toml

```diff
@@ -317,6 +318,12 @@ bench = false
 # reedline = { git = "https://github.com/nushell/reedline", branch = "main" }
 # nu-ansi-term = {git = "https://github.com/nushell/nu-ansi-term.git", branch = "main"}
 
+# this revision bumped terminal_size to use 0.4.0
+miette = { git = "https://github.com/zkat/miette.git", rev = "21e9a70ad8aacd1f32f515fa1540adc1c8df5341" }
+
+# removed rustix from non-unix builds
+terminal_size = { git = "https://github.com/cptpiepmatz/terminal-size.git" }
```

Comment:
Once this is ready to merge, this should ideally be merged back into `terminal-size` upstream (I saw your PR). Alternatively, this should have a `rev =` specification.

Reply:
I hope they take a look at my PR relatively soon.

Reply:
I believe miette and terminal_size will need to be published before we can release, right? I wonder if it's safe to land this without agreements from them to publish before Dec-24th.

Reply:
`miette` got updated on `main`, we only need the patch to `terminal-size` to go through, but the author doesn't seem to be very active